pkg/api/testapi/testapi.go

```diff
@@ -79,6 +79,10 @@ func init() {
 	Extensions = Groups[extensions.GroupName]
 }
 
+func (g TestGroup) ContentConfig() (string, *unversioned.GroupVersion, runtime.Codec) {
+	return "application/json", g.GroupVersion(), g.Codec()
+}
+
```

Comment:
It doesn't return a ContentConfig because of a circular dependency (an awkward one).  The intent was to ease initialization of the struct, especially for when we want to switch out the content type and codec together (to test protobuf and yaml and others)